keras/ops/numpy_test.py

```diff
@@ -3872,3 +3874,165 @@ def test_tri(self):
         self.assertAllClose(knp.Tri()(3), np.tri(3))
         self.assertAllClose(knp.Tri()(3, 4), np.tri(3, 4))
         self.assertAllClose(knp.Tri()(3, 4, 1), np.tri(3, 4, 1))
+
+
+class NumpyDtypeTest(testing.TestCase, parameterized.TestCase):
```

Comment:
This is the most useful class -- only through rigorous testing can we achieve consistency 👍